react/features/billing-counter/reducer.js

```diff
@@ -0,0 +1,29 @@
+import { ReducerRegistry } from '../base/redux';
```

Comment:
Looks like this might have been pulled in from upstream. We can either rebase your poll-feature branch so we don't see this diff or merge this in first and ensure the PR against upstream doesn't have any extra changes.